interface/HemisphereMixer.h

```diff
@@ -268,9 +268,9 @@ template <class EventClass> class HemisphereMixer : public BaseOperator<EventCla
         }
 
         std::vector<std::size_t> index_nns(knn_);
-        std::vector<double> dist_nns(knn_);
+        //std::vector<double> dist_nns(knn_);
```

Comment:
you have to reinitialize ev.best_match_distances_ vector to knn_ fixed size, because you will get a segfault otherwise.